options/locale/locale_nl-NL.ini

```diff
@@ -1,4 +1,4 @@
-app_desc=Een eenvoudige zelfgehoste Git service geschreven in Go
+app_desc=A painless self-hosted Git service.
 
```

Comment:
I see `A painless self-hosted Git service` as a slogan. In Ducth it's not common to translate slogans.